services/app-api/src/testHelpers/contractAndRates/rateHelpers.ts

```diff
@@ -110,32 +109,38 @@ const createRateRevision = (
     supportingDocuments: [
         {
             id: uuidv4(),
-            rateRevisionID: 'rateRevisionID',
+            rateDocumentRevisionID: 'rateDocRevisionID',
```

Comment:
shouldn't a document only ever have one parent ID?

Reply:
This is an artificial of having the named relationship. Here are the [docs](https://www.prisma.io/docs/concepts/components/prisma-schema/relations#disambiguating-relations). What do you think of this? Let me know your thoughts.